docs/config-file-ort-yml.md

```diff
@@ -125,11 +126,10 @@ scopes defined in the examples below match the scopes in your project.
 
 ### When to Use Curations
 
-Project-specific curations should be used when you want to correct the licenses detected in the source code of the
-project. If you need to correct the license findings for a third-party dependency then add a curation to
-[curations.yml](config-file-curations-yml.md) or [package configuration](config-file-package-configuration-yml.md).
+Project-specific license finding curations should be used when you want to correct the licenses detected in the source 
```

Comment:
'license finding curations' != curations

I believe this is out of scope of this PR.

Reply:
> 'license finding curations' != curations

I am aware.

The removed sentence: `If you need to correct the license findings for a third-party dependency then add a curation to
[curations.yml](config-file-curations-yml.md) or [package configuration](config-file-package-configuration-yml.md).`

Is now no longer true. As third-party dependencies can now also be curated in the `.ort.yml`.
I think this should be part of this PR.

Reply:
I meant "Project-specific license finding curations" don't exist, not even in this PR.

Reply:
> I meant "Project-specific license finding curations" don't exist, not even in this PR.

Project-specific license finding curations do exist as per the documentation: https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-ort-yml.md#curating-license-findings
Or at least a functionality is called like that in the documentation.

Are you talking about project-specific package configurations with a license finding curation?
Yes. This is out of the scope for this PR.

Reply:
> Are you talking about project-specific package configurations with a license finding curation?
Yes. This is out of the scope for this PR.

no, I'm talking about the the elements under
``
curations:
  license_findings:
``

Reply:
This was previously and still is documented:
`License finding curations should be used when you want to correct the licenses detected in the source code of the
project.`

Should I rephrase this to clarify this?